.github/workflows/test_fast.yml

```diff
@@ -20,11 +20,17 @@ jobs:
       fail-fast: false # Don't let a failed MacOS run stop the Ubuntu runs
       matrix:
         os: ['ubuntu-latest']
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+        python-version: ['3.7', '3.8', '3.10', '3.11']
```

Comment:
Is that change intentional?

Reply:
Yes, so we test non-UTC on one of the python versions